### ISSUE TITLE
Enable CrossValidation to test models

### ISSUE THREAD

**Issue by Froffri** [just-hms/DM-ML#10]
Invece che creare test set e training set potrebbe essere un'opzione migliore quella di usare la cross validation
